ui/app/components/secret-engine/configure-aws.ts

```diff
@@ -46,16 +48,33 @@ interface Args {
 export default class ConfigureAwsComponent extends Component<Args> {
   @service declare readonly router: Router;
   @service declare readonly store: Store;
+  @service declare readonly version: VersionService;
   @service declare readonly flashMessages: FlashMessageService;
 
   @tracked errorMessageRoot: string | null = null;
   @tracked errorMessageLease: string | null = null;
   @tracked invalidFormAlert: string | null = null;
   @tracked modelValidationsLease: ValidationMap | null = null;
+  @tracked accessType = 'iam';
+  @tracked disableAccessType = false;
```

Comment:
This doesn't need to be a tracked property if it's only set in the constructor! It could either be a getter, or not have `tracked` in front
```suggestion
  disableAccessType = false;
```
FYI: If you run into an error that's something like "you tried setting a value but you already set it in this lifecycle" then you may just need to remove the `= false` part. I can't remember the specifics of non-tracked values ATM

Reply:
TIL, thank you!